lib/views/general/_frontpage_articles.html.erb

```diff
@@ -0,0 +1,62 @@
+<% 
+    @research_articles = [
+    { title: 'Improving oversight of Access to Information',
+        url: 'https://research.mysociety.org/publications/improving-oversight',
+        img: 'paper-1.png',
```

Comment:
@garethrees not sure how to do the hotlinking would work. In this case I guess I could replace the img with the url
kind of like this: `img:https://research.mysociety.org/media/cache/57/75/57751fabb74b981cbe4e6f5d96d992b2@2x.png` the only problem is in a cache folder so not sure if this will keep working if we clear the cache.

Reply:
@ajparsons is there a permanent URL that exists for these research paper images?

Reply:
I don't think if this makes it easier or not, but there's a basic json API that would give the details based on a tag:

https://research.mysociety.org/embed/foi/limit:5/format:json/

Could define a new tag (which is how the 'transparency' page work on mysociety.org).

Alternatively I could probably spend 10 minutes adding a URL redirect in for consistent thumbnail images.

Reply:
> there's a basic json API that would give the details based on a tag

Oh, I did not know about this! 

We could use that pretty easily to construct the `@research_articles` list. That way we'd never have to update it (providing you maintain the API! 😁 )

```rb
@research_articles =
  JSON.parse(Net::HTTP.get(URI('https://research.mysociety.org/embed/foi/limit:10000/format:json/'))).
  with_indifferent_access[:items].map { |item| item.slice(:title, :url, :thumbnail) }
```

We'd want to handle a HTTP response/JSON parsing failure, but we've got a pattern for that [in AskTheEU for caching their blog](https://github.com/mysociety/asktheeu-theme/pull/44).

> Could define a new tag

`foi` is perfect!

> `limit:5`

I don't think we want to limit this. Is there a way to get everything, or is my `limit:10000` the best way?

Or we could just render the most recent, but I'd thought that a regular rotation might keep people interested…

Reply:
There seems to be a default limit, so `limit:1000` is probably right.

My sense is more recent things are *generally* more interesting than older things (quality control as much as anything and some things date), but can always review how it's doing later.

Reply:
Okay, I'll pull in most recent 10 and pick random 6, so that there's some variation but biases towards newest ✨